### PR TITLE
Fix bug with rendering of the "bamm:see" attributes

### LINKED ISSUE #196: [BUG] `bamm:see` rendered to multiple elements not annotated with it
**Describe the bug**
With `bamm aspect Test.ttl to svg > Test.svg` at bamm cli version 2.0.1, the following (correctly validated) sample input is rendered incorrectly:

```ttl
@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:1.0.0#>.
@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:1.0.0#>.
@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:1.0.0#>.
@prefix unit: <urn:bamm:io.openmanufacturing:unit:1.0.0#>.
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
@prefix : <urn:bamm:test:0.0.1#>.

:Test a bamm:Aspect;
    bamm:name "Test";
    bamm:properties (:prop1 :prop2);
    bamm:operations ();
    bamm:events ().

:prop1 a bamm:Property;
    bamm:name "prop1";
    bamm:characteristic :Enum1.
:prop2 a bamm:Property;
    bamm:name "prop2";
    bamm:characteristic :Enum2.

:Enum1 a bamm-c:Enumeration;
    bamm:name "Enum1";
    bamm:dataType xsd:string;
    bamm-c:values ("a" "b").
:Enum2 a bamm-c:Enumeration;
    bamm:name "Enum2";
    bamm:dataType xsd:string;
    bamm-c:values ("1" "2");
    bamm:see <https://test.com>.
```

As is clearly visible, `bamm:see` is only annotated at `:Enum2`. However, in the rendered SVG, both Enumerations have the link rendered:

image

Also, for some reason, timeseries related things are rendered.

**Where**
`bamm aspect Test.ttl to svg > Test.svg` at bamm cli version 2.0.1

**Additional Context**

This is, once again, also an issue in BAMM v2:

```ttl
@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#> .
@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#> .
@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#> .
@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix : <urn:bamm:test:0.0.1#> .

:Test a bamm:Aspect ;
   bamm:properties ( :prop1 :prop2 ) .

:prop1 a bamm:Property ;
   bamm:characteristic :Enum1 .

:prop2 a bamm:Property ;
   bamm:characteristic :Enum2 .

:Enum1 a bamm-c:Enumeration ;
   bamm:dataType xsd:string ;
   bamm-c:values ( "a" "b" ) .

:Enum2 a bamm-c:Enumeration ;
   bamm:see <https://test.com> ;
   bamm:dataType xsd:string ;
   bamm-c:values ( "1" "2" ) .
```

**Update**
This seems to not affect normal Characteristics, as can be seen here:

```ttl
@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix : <urn:bamm:test:0.0.1#> .

:Test a bamm:Aspect ;
   bamm:properties ( :prop1 :prop2 ) .

:prop1 a bamm:Property ;
   bamm:characteristic :Enum1 .

:prop2 a bamm:Property ;
   bamm:characteristic :Enum2 .

:Enum1 a bamm:Characteristic ;
   bamm:dataType xsd:string .

:Enum2 a bamm:Characteristic ;
   bamm:see <https://test.com> ;
   bamm:dataType xsd:string .
```

So potentially, this is an issue with this sparql subclause which is present at enumerations, but not characteristics in general:

https://github.com/OpenManufacturingPlatform/sds-sdk/blob/main/core/sds-aspect-model-document-generators/src/main/resources/diagram/bamm_1_0_0/enumeration2boxmodel.sparql#L108-L124

### PR DESCRIPTION
**Pull request by RaMisess**
Fixes #196

### CODE REVIEW

**Review by SteffenSta — APPROVED** [review]

**Review by waltersve — APPROVED** [review]
Looks good

File: core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test/1.0.0/AspectWithEnumOnlyOneSee.ttl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.openmanufacturing.test:1.0.0#>.
+
+:AspectWithEnumOnlyOneSee a bamm:Aspect;
+    bamm:name "Test";
```
Comment (atextor) [inline core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test/1.0.0/AspectWithEnumOnlyOneSee.ttl:22]: There is no `bamm:name` in BAMM 2.0.0 any more, please remove

**Review by atextor — CHANGES_REQUESTED** [review]

File: core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test/1.0.0/AspectWithEnumOnlyOneSee.ttl [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.openmanufacturing.test:1.0.0#>.
+
+:AspectWithEnumOnlyOneSee a bamm:Aspect;
+    bamm:properties (:prop1 :prop2);
+    bamm:operations ().
+
+:prop1 a bamm:Property;
+    bamm:name "prop1";
```
Comment (atextor) [inline core/sds-test-aspect-models/src/main/resources/valid/bamm_2_0_0/io.openmanufacturing.test/1.0.0/AspectWithEnumOnlyOneSee.ttl:26]: Please remove the `bamm:name` for all elements

**Review by atextor — CHANGES_REQUESTED** [review]

**Review by atextor — APPROVED** [review]
LGTM

**Pull request merged**